[skip ci] Update case

del-zhenwu opened the pull request:
Signed-off-by: zhenwu <zhenwu@milvus.io>

Commit:
update index case

Signed-off-by: zhenwu <zhenwu@milvus.io>

Commit:
add index cases

Signed-off-by: zw <zw@milvus.io>

Commit:
merge from mater

Signed-off-by: zw <zw@milvus.io>

yanliang567 commented on tests20/python_client/base/index_wrapper.py:
```diff
@@ -10,41 +10,33 @@
     index = None
 
-    def index_init(self, collection, field_name, index_params, name="", check_res=None, check_params=None, **kwargs):
+    def init_index(self, collection, field_name, index_params, name="",check_task=None, check_params=None, **kwargs):
```
check_params -> check_items, and we shall pass the check items and expectations into the check_items

yanliang567 commented on tests20/python_client/testcases/test_index.py:
```diff
@@ -72,9 +80,8 @@ def test_index_collection_None(self):
-        log.error(str(ex))
-        assert "invalid" or "illegal" in str(ex)
+        self.index_wrap.init_index(None, default_field_name, default_index_params, name=index_name, check_task=CheckTasks.err_res,
+                                           err_code=1, err_msg="Collection")
```
we shall pass the check items and expectations into the check_items:
check_items={err_code: 1, err_msg: "blablabla..."}

Commit:
update index case

Signed-off-by: zw <zw@milvus.io>

Commit:
update utility cases

Signed-off-by: zw <zw@milvus.io>

Commit:
[skip ci]
Merge branch 'master' of https://github.com/milvus-io/milvus into update_case_2

Commit:
[skip ci] update index cases

Signed-off-by: zw <zw@milvus.io>

yanliang567 reviewed (approved):
/lgtm
/approve

yanliang567 commented:
/ci-pass
